### PR TITLE
[dev] AB#51574: MUI Skeleton, List Fixes, Button Fixes, Popover Update, and ActionBar Fix

### PR DESCRIPTION
**Pull request by cbrewer1689**

### CODE REVIEW

**Comment by cbrewer1689** [comment]
Do not merge until PR #327 is merged.

File: package-lock.json
Comment (groberts314) [inline package-lock.json:1905]: There are a lot of removals from `package-lock.json`, which seems odd since we _added_ a new dependency to `package.json`.  Obviously not a blocker and not necessarily actionable, but seems possibly a bit strange.

**Review by groberts314 — APPROVED** [review]
LGTM

I had to look up _what the heck_ a "Skeleton" was! =)
https://material-ui.com/components/skeleton/

File: package-lock.json
Comment (cbrewer1689) [reply package-lock.json]: Let me take a look.

Comment (cbrewer1689) [reply package-lock.json]: Should look a lot better now.

Comment (groberts314) [reply package-lock.json]: Yeah, _MUCH_ better.  Curious, what was the issue.  But in any event, _Resolved!_

File: src/dataDisplay/divider/divider.jsx
Comment (groberts314) [inline src/dataDisplay/divider/divider.jsx:42]: **_Not a Blocker_** should we put a documentation comment on `designVersion` prop?

File: src/dataDisplay/list/list.jsx
Comment (groberts314) [inline src/dataDisplay/list/list.jsx:43]: Same NAB here =)

File: src/dataDisplay/divider/divider.jsx
Comment (cbrewer1689) [reply src/dataDisplay/divider/divider.jsx:46]: Updated.

File: src/dataDisplay/list/list.jsx
Comment (cbrewer1689) [reply src/dataDisplay/list/list.jsx:47]: Updated.

File: package-lock.json
Comment (cbrewer1689) [reply package-lock.json]: Mmm, probably switching back and forth between branches and maybe not doing the proper `npm install`. That's my guess.

Comment (groberts314) [reply package-lock.json]: Yeah, makes sense.

File: src/dataDisplay/divider/divider.jsx
Comment (groberts314) [reply src/dataDisplay/divider/divider.jsx:46]: Looks good.  Resolving.

File: src/index.js
Comment (groberts314) [inline src/index.js:64]: Why is this export necessary?  Just curious?  It's already available as `<Page.ActionBar>`, so this makes it available as `<PageActionBar>` as well, right?

File: src/dataDisplay/list/list.jsx
Comment (groberts314) [reply src/dataDisplay/list/list.jsx]: Looks good.  Resolving.
(Note: this is now in `src/dataDisplay/list/listBase.jsx`)

**Pull request merged**
